client/api/repository.js

```diff
@@ -39,11 +41,23 @@ function publishRepositoryMeta(id) {
   return request.post(urls.publish(id)).then(res => res.data);
 }
 
+function addTag(data) {
```

Comment:
Might be nicer if you destructure it here to see what you expect within data (since it is failry simple). Thoughts ? 🤔